tfdml/kernels/dml_one_hot_op.cc

```diff
@@ -220,17 +220,14 @@ class DmlOneHotKernel : public DmlKernel
             indices = dml::Cast(indices, DML_TENSOR_DATA_TYPE_UINT32);
         }
 
+        // TFDML #40634243
```

Comment:
Can you also put a TODO? So it's easier to find in the future. And maybe also put the title of the bug here.